From 4317d43d74996ad7ec3270b1e786e91a19312309 Mon Sep 17 00:00:00 2001
From: Adam Duskett <Adamduskett@outlook.com>
Date: Sat, 24 Jun 2017 13:08:57 -0400
Subject: [PATCH] remove test -z "$DESTDIR" from ltmain.sh.

Without this patch, buildroot-libtool-v2.4.patch will fail to apply.

Signed-off-by: Adam Duskett <Adamduskett@outlook.com>
---
 ltmain.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ltmain.sh b/ltmain.sh
index 5d29bd6..6c90601 100644
--- a/ltmain.sh
+++ b/ltmain.sh
@@ -6658,7 +6658,7 @@ func_mode_link ()
 	  *)
 	    if test "$installed" = no; then
 	      func_append notinst_deplibs " $lib"
-	      test -z "$DESTDIR" && need_relink=yes
+	      need_relink=yes
 	    fi
 	    ;;
 	  esac
-- 
2.9.4

